### ISSUE TITLE
Trying to use it

### ISSUE THREAD

**Issue by casey1701** [AxisCommunications/media-stream-player-js#31]
I'm trying to use it with an Axis Q line.  Loading it just like the example says:

```
<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>media-stream-player</title>
    <script src="media-stream-player.min.js"></script>
  </head>

  <body>
    <div id="root" />
    <media-stream-player hostname="166.168.152.177" autoplay />
  </body>
</html>

```
It pops up and you can see the player for a second and then it all disappears.

**Comment by lekoaf** [comment]
Do you get any errors in the console?

**Comment by lekoaf** [comment]
Have you tried the react-app example as well?

**Comment by casey1701** [comment]
I haven't tried the react app. I'm not set up to do that. The IP I put in my code up there is a camera available to the public. I can see connections to the camera open from the network tab, and if I highlight my mouse over it I can see an image.

What I would really love to be able to do is embed a live stream and have the crosshair joystick work for PTZ control. Do you know of any way to do that?

**Comment by lekoaf** [comment]
Try adding the 'autoplay' property to your media-stream tag.

Sure, it's possible to hook up the joystick and control the PTZ. It would require a custom coding solution though. Maybe using the [GamePad](https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API/Using_the_Gamepad_API) API to send requests to the PTZ cgi on the camera.

**Comment by lekoaf** [comment]
I can not connect to http://166.168.152.177. Are you sure it's working properly?

**Comment by lekoaf** [comment]
Yeah, that camera is either not public, isn't set up properly or doesn't work at all.

```bash
GET http://166.168.152.177/axis-cgi/usergroup.cgi net::ERR_CONNECTION_REFUSED
TypeError: Failed to fetch
Error: websocket connection timed out
WebSocket connection to 'ws://166.168.152.177/rtsp-over-websocket' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
```

**Comment by casey1701** [comment]
Oh sorry! I don't think that one turns on for another 30 minutes. Can you
try 166.168.152.177? Thanks s much for trying it.

Oh I didn't mean an actual joystick. I was talking about the axis feature.
That's what they call it when the crosshair is on the middle of the video
and you can drag it.

**Comment by lekoaf** [comment]
I tried a different IP and got it work without a problem with the react example. It works because it runs on a localhost webserver.

If I try the webcomponent example using the `file:///` protocol it fails with this warning:

```
A cookie associated with a cross-site resource at http://166.168.152.176/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
```

I suggest trying to get the react-exmple running instead. Or uploading it to some webserver and try that.

Regarding the joystick & crosshair. You could combine this library with [media-stream-overlay](https://github.com/AxisCommunications/media-overlay-library-js) and create a draggable crosshair which sends events / requests to the PTZ cgi.

**Comment by lekoaf** [comment]
Oh wait. I actually got the webcomponent-example to work as well. Seems like there is some CSS issue with height / width of the html. I'll look in to that. Try this:

![css](https://user-images.githubusercontent.com/7765599/88389871-a211b480-cdb7-11ea-9cbf-b08e806ecfb9.jpg)

**Comment by lekoaf** [comment]
Fixed in #56 .
